src/react/workflow/TransitionTable.tsx

```diff
@@ -67,11 +74,15 @@ const TransitionTable = ({
     transitionWithDesciption.push({
       triggerDelayDays,
       locationName,
-      description: `Objects older than ${triggerDelayDays} day(s) will transition to ${locationName} `,
+      description: `Objects older than ${triggerDelayDays} day${
+        typeof triggerDelayDays === 'string' && parseInt(triggerDelayDays) > 1
```

Comment:
parseInt(triggerDelayDays, 10) // because js and browser implementation... if triggerDelayDays = '08' => base 8